elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -111,7 +127,55 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
We might want to organize the dependencies a bit more.  Break them into test and non-test sections.  Also, dependencies from the same group should be adjacent to one another.

Reply:
Tried to fix it as best as I can, if there's still things wrong let me know and I can fix them.